develop/plone/content/types.rst

```diff
@@ -17,7 +17,7 @@ Plone has two kind of content types subsystems:
 
 * :doc:`Dexterity </develop/plone/content/dexterity>`-based (new)
 
-* See also Plomino (later in this document).
+See also Plomino (later in this document).
```

Comment:
For the ones only looking at the diff, above the list there is a sentence saying *Plone has two kind of content types*, well, the list had **three** items :-)